lib/logproto/logproto-server.h

```diff
@@ -155,6 +177,13 @@ void log_proto_server_free(LogProtoServer *s);
     .construct = prefix ## _server_plugin_construct,  \
   }
 
+#define LOG_PROTO_SERVER_PLUGIN_WITH_GRAMMAR(parser_func, __name) \
+  {             \
+    .type = LL_CONTEXT_SERVER_PROTO,            \
+    .name = __name,         \
+    .parser = &parser_func,  \
```

Comment:
The `&` operator is redundant here.

Reply:
Fixed. The name was invalid: parser is not a function pointer, it's a CfgParser instance.